wiki/Gameplay/Judgement/osu!/fr.md

```diff
@@ -0,0 +1,40 @@
+---
+no_native_review: true
+needs_cleanup: true  # see the original article
+---
+
+# osu! judgement system
```

Comment:
if english headers are going in as-is for now then we should slap `no_native_review` on all applicable translations

Reply:
just `outdated` is enough imo

Reply:
perhaps